prow/images/cleaner/Makefile

```diff
@@ -0,0 +1,13 @@
+IMG_NAME = cleaner
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(IMG_NAME)
+TAG = $(DOCKER_TAG)
```

Comment:
see how k8s is doing it: https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/Makefile